lib/datadog_api_client/configuration.rb

```diff
@@ -230,6 +244,13 @@ def self.default
       @@default ||= Configuration.new
     end
 
+    def backoff_base=(value)
+      if value < 2
+        raise ArgumentError, 'backoff_base cannot be smaller than 2'
```

Comment:
May need a native speaker, but I would say `less than`

Reply:
I think smaller than and less then are synonyms. Both works